src/polodb_core/vm/vm.rs

```diff
@@ -827,6 +829,60 @@ impl VM {
                         self.pc = self.pc.add(1);
                     }
 
+                    DbOp::Regex => {
+                        let val1 = &self.stack[self.stack.len() - 2];
+                        let val2 = &self.stack[self.stack.len() - 1];
+
+                        self.r0 = 0;
+
+                        if let Bson::RegularExpression(re) = val2 {
+                            let mut re_build = RegexBuilder::new(re.pattern.as_str());
+                            for char in re.options.chars() {
```

Comment:
@vincentdchan Should I set a limit to the max number of character inside the `options` string ? The max would be 6 chars. This would avoid the user passing something like `options: "imsuUxxu"` <= duplicate characters

Reply:
I am not quite sure about this. Have you tested the behavior of MongoDB?